neon_core/__init__.py

```diff
@@ -22,22 +22,29 @@
 # LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 # NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE,  EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-from os.path import join, dirname, exists
+
+from os.path import join, dirname
 import xdg.BaseDirectory
 import json
 from ovos_utils.json_helper import merge_dict
 from ovos_utils.system import set_root_path
 from ovos_utils.configuration import set_config_name
 
+from neon_utils import LOG
+from neon_utils.configuration_utils import write_mycroft_compatible_config
+
 NEON_ROOT_PATH = dirname(dirname(__file__))
 
-# first thing to do is ensure user holmes.conf exists
-# NOTE: must be done before any mycroft (neon) import
-def setup_holmes_config():
-    HOLMES_CONFIG = join(xdg.BaseDirectory.save_config_path("HolmesV"),
-                         "holmes.conf")
 
-    _NEON_HOLMES_CONFIG = {
+def setup_ovos_core_config():
+    """
+    Runs at module init to ensure base holmes.conf exists to patch ovos-core. Note that this must run before any import
```

Comment:
holmes.conf in docstr should be ovos.conf

Reply:
good catch